astra/src/main/java/com/slack/astra/logstore/search/LogIndexSearcherImpl.java

```diff
@@ -79,6 +84,9 @@ public void afterRefresh(boolean didRefresh) {
 
     // initialize the adapter with whatever the default schema is
     openSearchAdapter.reloadSchema();
+    allowIncludeAndExcludeSource =
```

Comment:
Since this is a net-new optional field, is there any risk to just shipping support without wrapping it in a boolean check? As the only folks theoretically invoking this branch are those explicitly requesting something with an include/excludes source?